### DISCUSSION TITLE
Unresponsive on archive calls when forking after long time

### DISCUSSION THREAD

**Discussion by JasperTimm** [ConsenSys-archive/ganache#2588] [Q&A]
Hi there,

I'm running a local fork of mainnet, using the default provider `--fork.network "mainnet"`, with ganache 7.0.2.

It seems like after a day or so of the ganache process running, calls which require the use of the archive node become unresponsive, i.e. the RPC call is sent, logged, but no response is returned. Any calls which don't require it, like the balance of accounts created at genesis return fine. There doesn't appear to be any abnormal resource usage.

The same issue happens when specifying a custom archive node.

I thought perhaps it might be related to this? #2253 

Is there a connection to the node going stale and a need for it to be refreshed maybe?

**Comment by davidmurdoch** [comment]
Hm. When you use your own custom archive node are you using a Websocket URL or HTTP?

**Reply by JasperTimm** [reply]
So, some somewhat rough feedback on this - 
- I think whenever I've used the default '--fork.network "mainnet"' it's become unresponsive 100% of the time after a day or so.
- From memory, the same thing happened when I used the HTTP URL for my own archive node (Alchemy)
- I've been using the WSS URL from Alchemy for about 3 days now and there doesn't seem to be an issue at all. All requests are still being responded to so far.

Does that help? I'm happy to try to give more info.

**Reply by davidmurdoch** [reply]
I doesn't help, but I very much appreciate the response! Hahaha Do you want to open an issue for this (so you can be notified of progress -- if not, I don't mind opening it myself)? I think the first step would be for us to add some additional logging information specifically to track down what's happening here.

**Reply by JasperTimm** [reply]
No worries, issue created: #2628
